src/cli/cli.rs

```diff
@@ -74,8 +74,22 @@ impl Debug for dyn SubCommand {
 /// Trace packets on the Linux kernel
 ///
 /// packet-tracer is a tool for capturing networking-related events from the system using ebpf and analyzing them.
-#[derive(Args, Default, Debug)]
-pub(crate) struct MainConfig {}
+#[derive(Args, Debug, Default)]
+pub(crate) struct MainConfig {
+    #[arg(
+        long,
+        default_value = "warn",
+        help = "Log level: error, warn, info, debug. Default: warn."
```

Comment:
Default warn makes the tool really silent (hopefully we don't have warnings :-D).
Not even "hey we are capturing" or the like.
How about default=info?

Reply:
Also, consider using clap's PosibleValueParser